ios/Classes/PdftronFlutterPlugin.h

```diff
@@ -202,6 +203,7 @@ static NSString * const PTXfdfCommandArgumentKey = @"xfdfCommand";
 static NSString * const PTXfdfArgumentKey = @"xfdf";
 static NSString * const PTBookmarkJsonArgumentKey = @"bookmarkJson";
 static NSString * const PTPageNumberArgumentKey = @"pageNumber";
+static NSString * const PTTitleArgumentKey = @"title";
```

Comment:
```suggestion
static NSString * const PTBookmarkTitleArgumentKey = @"bookmarkTitle";
```

Reply:
renamed key, kept string for consistency with Android